addon/src/main/java/com/vaadin/flow/component/charts/ChartOptions.java

```diff
@@ -0,0 +1,103 @@
+package com.vaadin.flow.component.charts;
+
+import com.vaadin.flow.component.ComponentUtil;
+import com.vaadin.flow.component.UI;
+import com.vaadin.flow.component.charts.model.AbstractConfigurationObject;
+import com.vaadin.flow.component.charts.model.Lang;
+import com.vaadin.flow.component.charts.util.ChartSerialization;
+
+import elemental.json.JsonObject;
+import elemental.json.impl.JreJsonFactory;
+
+/**
+ * The ChartOptions configures a page local global options like localized texts
+ * for charts.
+ */
+public class ChartOptions extends AbstractConfigurationObject {
+
+    private Lang lang;
+    private transient JreJsonFactory jsonFactory;
+
+    protected ChartOptions() {
```

Comment:
Is the idea to prevent ChartsOptions from being instantiated unless it is extended?

Reply:
Extending would also require overriding the get(ui) method, but still wouldn't want to close the option